hugo/layouts/partials/home/our-links.html

```diff
@@ -1,10 +1,6 @@
-<p>
-    {{ $home := $.Site.Data.home.home }}
-    {{ range where $home.hero.links "url" "ne" "#"  }}
```

Comment:
The old approach lets the owner enable/disable a link, just setting the `url` value to `#` or with a normal url.
It was used in the past, and it could be useful in the future.